requirements.yaml

```diff
@@ -4,4 +4,4 @@
   version: v0.5.4
 - src: https://github.com/metal-stack/metal-roles.git
   name: metal-roles
-  version: v0.2.2
+  version: fb3f338010c534629220433f2781e70343f8beaa
```

Comment:
this needs to point to a release

Reply:
merged @kolsa 's branch alloc-grpc, which allows us to specify v0.3.0 for metal-roles